contributors/design-proposals/pod-priority-resourcequota.md

```diff
@@ -75,5 +75,51 @@ Unlike pod spec, ResourceQuota does not have Priority `*int32` in its spec. As e
 ### Expected behavior of ResourceQuota admission controller
 
 * We do not enforce referential integrity across objects. i.e. Creation or update of `ResourceQuota` object which new `ResourceQuota` names a `PriorityClass` that does not exist, and empty `PriorityClassNameList` field are all allowed
-* It is configurable for `ResourceQuota` admission controller to reject pod with priority set but has no matched ResourceQuota found or not.ß
 * `ResourceQuota` admission controller **should not reject** deletion of `ResourceQuota` objects with reference.
+
+#### Changes in AdmissionConfiguration
+
+It is configurable for `ResourceQuota` admission controller to reject pod with priority set but has no matched `ResourceQuota` found or not. 
+
+This requires an extra boolean option in `LimitedResource` type:
+```go
+// LimitedResource matches a resource whose consumption is limited by default.
+// To consume the resource, there must exist an associated quota that limits
```

Comment:
@derekwaynecarr 
This comment in the existing code says that when there is a matching LimitedResource, there must exist an associated quota to consume the resource.
If this means that absence of quota for limited resources means no quota, I think we should keep the existing behavior and do not need to change it.

Reply:
According to previous discuss, I think Derek would prefer to make this configurable.

Reply:
Making it configurable is fine with me, but we shouldn't change default behavior. If default behavior was no quota when there was no ResourceQuota, it should remain the same after making it reconfigurable. That means, when the configuration is missing, no ResourceQuota at a priority should mean no quota at that priority.

Reply:
The default behavior is if I have no quota, then I can consume an unbounded amount of the resource.

If I mark a resource as limited by default in quota configuration, then the inverse becomes true.  I must have a quota in order to consume the resource.

Reply:
Yes, that's what I meant. We should keep the same defaults.